analysis/reports/pf_report.Rmd

```diff
@@ -0,0 +1,236 @@
+---
+title: "Pharmacy First"
+output:
+  html_document:
+    toc: true
+    toc_depth: 4
+    theme: journal
+date: "01-01-2020 through 31-06-2024 by month"
+---
+```{css, echo=FALSE}
+pre {
+  max-height: 300px;
+  overflow-y: auto;
+}
+pre[class] {
+  max-height: 200px;
+}
+```
+
+<style>
+  body {
+    text-align: justify;
+     font-size: 11pt}
+</style>
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+
+library(tidyverse)
+library(here)
+library(readr)
+
+
+```
+<br>
+
+## Background <a name="background"></a>
+
+Add background here.
+
+<br>
+
+## Methods <a name="methods"></a>
+
+This study used data from OpenSAFELY-TPP, which covers 40% of the population of England. For a description of the representativeness of this sample, please see our manuscript [here](https://wellcomeopenresearch.org/articles/7-191/v1). Individuals were included if they were alive and registered at a TPP practice each month, across the study period. Patients were excluded if their listed age was not between 0 and 120 years.
+
+Counts represent patients with at least one clinical code of relevance in that month. Patients with more than one of the same clinical code in a month were only counted once. Rates divide the count by the included study population and multiply by 1,000 to achieve a rate per 1,000 registered patients.
+
+Counts <=7 have been redacted and all numbers rounded to the nearest 5 to avoid potential re-identification of individuals. The rates displayed were computed with these rounded counts.
+
+Our data relies on a relevant Pharmacy First code being added to a patient's GP record. The Pharmacy First service relies on [GP Connect: Update Record](https://digital.nhs.uk/services/gp-connect/gp-connect-in-your-organisation/gp-connect-update-record) to update a patient's GP record with consultation information from the community pharmacy. Following the launch of the Pharmacy First service, there has been a [gradual roll-out of GP Connect: Update Record](https://cpe.org.uk/our-news/gp-connect-update-record-rollout-and-flow-of-information/) across the approved community pharmacy IT system suppliers.
+
+*Need to complete this*
+
+Links to the codelist for each analysis can be found beneath the relevant section.
+
+This report contains the following sections:
+
+* [Clinical Services](#services)
+* [Clinical Pathways](#pathways)
+  * [Clinical condition](#condition)
+  * [Days of the week](#dotw)
+  * [NHS region](#region)
+  * [Ethnicity](#ethnicity)
+  * [IMD](#imd)
+  * [Shingles](#shingles)
+    + [Sex](#shingles_sex)
+    + [Age](#shingles_age)
+    + [Ethnicity](#shingles_ethnicity)
+    + [IMD](#shingles_imd)
+    + [Region](#shingles_region)
+    + [Medication supply](#shingles_supply)
+  * [Urinary tract infection etc.](#urt)
+* [Hypertension](#hyp)
+  * [Age](#hyp_age)
+  * [Sex](#hyp_sex)
+  * [Region](#hyp_region)
+* [Contraception](#contra)
+  * [Age](#contra_age)
+  * [Region](#contra_region)
```

Comment:
@viv3ckj do you remember where this came from? I think Chris had a Jupyter notebook but I cant find it anywhere?